app/services/cocktail_service.rb

```diff
@@ -0,0 +1,16 @@
+class CocktailService
+  def self.get_cocktail_details(cocktail_id)
+    response = Faraday.get('https://www.thecocktaildb.com/api/json/v1/1/lookup.php?') do |req|
+      req.params['i'] = cocktail_id
+      req.params['api_key'] = ENV['cocktail_key']
+    end
+
+    parse_json(response)['drinks'].first
+  end
+
+  private
+
+  def self.parse_json(response)
+    JSON.parse(response.body, sybmolize_names: true)
+  end
```

Comment:
Might want to consider making this into a module so we can use it in tests/other environments as well?

Reply:
^ Also a refactor option, not critical for this PR